example/gardener-local/kind/cluster-local.yaml

```diff
@@ -12,6 +12,8 @@ nodes:
   - hostPath: example/gardener-local/controlplane
     containerPath: /etc/gardener/controlplane
     readOnly: true
+  - hostPath: dev/local-backupbuckets
+    containerPath: /etc/gardener/local-backupbuckets
```

Comment:
Question: will this approach work with a multi-node kind cluster? 
So far we don't have a use case for it, but it might come soon. Currently, there should be no restriction for adding more nodes to kind. But if this change imposes this restriction, we might need to think about the approach again.

Reply:
I think it should be ok. This can be set on all nodes to point to the same directory on the local machine. Then any shoot etcd that gets scheduled on any of the nodes would also be able to write/read from the same directory. 
And since the path for the backups would look like `local-backupbuckets/<bucket-id>/<backup-entry-name>` there should be no name collisions.